CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+## [Unreleased - 1.0.4]
```

Comment:
`Unreleased` does not really cover it. If you want to write history, it is okay to include at least the first official release `1.0.0`. See earlier release notes here: https://github.com/thehyve/podium/releases.

Reply:
To be clear, `Unreleased` does not represent the previous versions, but simply marks that 1.0.4 is not released yet (this mark should be removed later).

Reply:
Ah, of course :)